### DISCUSSION TITLE
【求助】如果有人在这里发煌片怎么办？

### DISCUSSION THREAD

**Discussion by LLLCCCCwdadasdzxczxcz** [share121/inter-knot#1722] [其他]
如果有人在这里发煌片怎么办？

**Comment by lllliufeng** [comment]
可以举办口牙

**Comment by capacitor1** [comment]
黄片来喽！！！
![image](https://github.com/user-attachments/assets/8c6ad216-eb0f-43ab-bcb6-294540ab73cc)
![image](https://github.com/user-attachments/assets/92c3da5f-a6b0-48c6-98d9-fdd28f5c9e13)
![image](https://github.com/user-attachments/assets/2c62f3e7-b181-4b3a-a4f0-2d0a56fe9de1)
你就说黄不黄吧doge

**Reply by fu-fu404** [reply]
太黄了，我要举办你

**Comment by lllliufeng** [comment]
还真是
狠活，举办了（
